#Publish all joint states
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
  extra_joints:
    - name: 'finger_joint'
      position: 0.6

# eTaSl example controller
etasl_controller_cmd:
  type: "etasl_ros_controllers/EtaslController"
  joints:
    - joint_a1
    - joint_a2
    - joint_a3
    - joint_a4
    - joint_a5
    - joint_a6
  input:
    names:
      - "speed"
      - "peg1_frame"
      - "peg2_frame"
      - "peg3_frame"
      - "peg4_frame"
      - "peg5_frame"
      - "block_frame"
      - "netft_data"
    types:
      - "Scalar"
      - "Frame"
      - "Frame"
      - "Frame"
      - "Frame"
      - "Frame"
      - "Frame"
      - "Wrench"
  task_specification: "$(find etasl_ros_control_cmd)/scripts/peg_in_hole.lua"